pkgs/by-name/cr/cro-mag-rally/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "CroMagRally";
+  version = "unstable-2023-05-21";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
```

I do not think the repository name that we happen to pull the source from should depend on the package name. Removing this removes the need for the `rec` statement as well. The unstable version should also include the last published version: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

```suggestion
stdenv.mkDerivation {
  pname = "CroMagRally";
  version = "3.0.0-unstable-2023-05-21";

  src = fetchFromGitHub {
    owner = "jorio";
    repo = "CroMagRally";
```